README.md

```diff
@@ -3,7 +3,7 @@ shopify-sinatra-app [![Build Status](https://travis-ci.org/kevinhughes27/shopify
 
 "A classy shopify app"
 
-shopify-sinatra-app is lightweight extension for building Shopify apps using Sinatra. It comes with the Shopify API gem for interacting with the Shopify API and uses the Shopify omniauth gem to handle authentication. The framework itself provides a handful of helper methods to make creating your app as easy as possible. The framework was designed with deployment to Heroku in mind and following the instructions below I've been able to create a new application from scratch, deploy it to Heroku and install on my live test shop in less than 5 minutes.
+shopify-sinatra-app is lightweight extension for building Shopify apps using Sinatra. It comes with the Shopify API gem for interacting with the Shopify API and uses the Shopify omniauth gem to handle authentication (it does not support Private App token based auth). The framework itself provides a handful of helper methods to make creating your app as easy as possible. The framework was designed with deployment to Heroku in mind and following the instructions below I've been able to create a new application from scratch, deploy it to Heroku and install on my live test shop in less than 5 minutes.
 
```

Comment:
I'm happy to change the words.. but my struggle as a new user was that it wasn't explicit to me that auth differed between Partner and Private. It's there in the docs but there are _many_ _many_ words between these repos, the api client and the knowledge base. Explicitly calling out Private Apps as not supported IMO is a good idea.

How about:
`handle authentication via Oauth (other auth methods, such as Private Apps, are not supported)`?